tests/task_t_1.spec.js

```diff
@@ -4,4 +4,10 @@ test('basic test', async ({ page }) => {
   await page.goto('https://playwright.dev/');
   const title = page.locator('.navbar__inner .navbar__title');
   await expect(title).toHaveText('Playwright');
+});
+
+test('duck duck go T 1', async ({ page }) => {
```

Comment:
Test should be named in a way which would explain what it is checking, e.g. "'Checks that duckduckGo page can be opened"